script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt

```diff
@@ -904,7 +905,7 @@ But one would do well to stand in awe of the majesty of the planet.
 -- Page 328, Offset 39313.
 -- 人智における最高峰の大魔術を受け、なお月の姫は倒れない。
 // First 月の姫??
-For even the pinnacle of grand magecraft devised by man cannot yet fell the princess of the moon.
+For even the pinnacle of grand magecraft devised by man cannot yet fell the Princess of the Moon.
```

Comment:
Not sure if this should be caps, again defer to Petrikow

Reply:
I thought it's a title like Princess of the True Ancestors. A little help then? @Petrikow

Reply:
It's not something that shows up particularly often.
Think it's used by Roa in MBAA and MBTL at points.
I guess it does sort of sound like a title, so why not capitalize it

Reply:
caps it is then